mtgtools/PCardList.py

```diff
@@ -555,104 +555,88 @@ def unique_cards(self):
         """
         return PCardList(list(set(self.cards)))
 
+    def of_type(self, card_type: str, **kwargs):
```

Comment:
At this point it would be better to leave out the typehint just for consistency. It might also bring some backwards compatibility issues for people who are using some older python version